serverless/src/generateNotebook/notebookTemplate.ipynb

```diff
@@ -0,0 +1,379 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9955771c-6d23-493d-b783-dc158f834584",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Search for the granule by the name and provider\n",
+    "search_result = earthaccess.search_data(granule_name=\"20241017090000-JPL-L4_GHRSST-SSTfnd-MUR-GLOB-v02.0-fv04.1\", provider=\"POCLOUD\")\n",
+    "\n",
+    "# Get the data for the granule using earthaccess.open\n",
+    "files_array = earthaccess.open(search_result)\n",
+    "\n",
+    "# Get the first file in the array which will be the granule\n",
+    "file = files_array[0]\n",
+    "\n",
+    "file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1b6ec5ad-fcfa-449b-af50-a8ad29a56d4f",
+   "metadata": {},
+   "source": [
+    "## Open the dataset using `xarray.open_datatree()`\n",
+    "\n",
+    "After locating and downloading the data, you can now use `xarray.open_datatree()` to open the dataset created by  `earthaccess`. `xarray.DataTree` is a class available in `xarray`, that enables the working with [hierarchical data](https://docs.xarray.dev/en/stable/user-guide/hierarchical-data.html). Each group in an input file has a node, where variables within that group are represented using the standard `xarray.Dataset` class.\n",
+    "\n",
+    "Find more information about `xarray.open_datatree()` and its parameters in the [xarray.open_datatree documentation](https://docs.xarray.dev/en/stable/generated/xarray.open_datatree.html)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1da22567-9e35-446e-b237-6468046bbc53",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Open the dataset with xarray\n",
+    "dt = xr.open_datatree(\n",
+    "    file, # The granule dataset from earthaccess\n",
+    "    engine='h5netcdf', # The engine requried to open NetCDF files\n",
+    ")\n",
+    "\n",
+    "dt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "508dcd76-0e18-4f37-ba4f-dd0466ddc7cb",
+   "metadata": {},
+   "source": [
+    "## Select a subset of the data using `xarray.DataTree.sel()`\n",
+    "\n",
+    "The `xarray.DataTree.sel()` function can be used to return a new dataset which has been indexed to a specific bounding area. For large datasets, this can result in improved performance when doing analysis and plotting. \n",
+    "\n",
+    "Find more information about `xarray.DataTree.sel()` and its parameters in the [xarray.DataTree.sel documentation](https://docs.xarray.dev/en/latest/generated/xarray.DataTree.sel.html)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a091448e-6cdb-4ab0-a8c5-5d61648f1fb0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define the bounding area\n",
+    "\n",
+    "# Select the data within the bounding box applied in Earthdata Search at the time of generation.\n",
```

Comment:
This section isnt quite right. I think we can make it a little more useful for the user.

When the user has a bounding box applied, the section should read:

```
# Select the data within the bounding box that was applied in Earthdata Search.
min_lon = -92.17969
min_lat = 22.19104
max_lon = -80.89453
max_lat = 31.12491

# To select all of the data for the granule, remove the variables above and uncomment the following coordinate variable declarations.
# min_lon = -90
# min_lat = -180
# max_lon =  90
# max_lat =  180
```
When a custom bounding box is **not** applied, the section should read:

```
# Select the data all data by setting coordinate variable declarations for the entire globe. These values can be updated to subset the data to a smaller area of interest. The values can be set manually or by using a bounding box before generating a notebook in Earthdata Search.
min_lon = -90
min_lat = -180
max_lon =  90
max_lat =  180
```